<h1>Exercise 5.1 - Hasan Haq</h1>

<b>In the BRFSS (see Section 5.4), the distribution of heights is roughly normal with parameters μ = 178 cm and σ = 7.7 cm for men, and μ = 163 cm and σ = 7.3 cm for women.

In order to join Blue Man Group, you have to be male between 5’10” and 6’1” (see http://bluemancasting.com). What percentage of the U.S. male population is in this range? Hint: use scipy.stats.norm.cdf.</b>

In [3]:
# Import the modules needed to complete this exercise

from __future__ import print_function, division

import numpy as np
import nsfg
import thinkstats2
import thinkplot
import scipy.stats

In [40]:
# Use thinkstats2.EvalNormalCDF function to calculate % of male population between 5'10" and 6'1"

a = thinkstats2.EvalNormalCdf(177.8, mu=178, sigma=7.7)
b = thinkstats2.EvalNormalCdf(185.4, mu=178, sigma=7.7)

print("% of population <= 5'10:", "{0:.2f}%".format(a*100))
print("% of population <= 6'1:", "{0:.2f}%".format(b*100))
print("% of population between 5'10 and 6'1:", "{0:.2f}%".format((b-a)*100))

% of population <= 5'10: 48.96%
% of population <= 6'1: 83.17%
% of population between 5'10 and 6'1: 34.21%


# The percentage of population of US males between 5'10" and 6'1" is 34.2%